homeassistant/components/gpstracker/device_tracker.py

```diff
@@ -0,0 +1,163 @@
+"""Platform for gpstracker.device_tracker integration."""
+from __future__ import annotations
+
+import asyncio
+from datetime import timedelta
+import uuid
+
+import gps_tracker
+from gps_tracker import AsyncClient, Tracker
+from gps_tracker.client.datatypes import Tracker01, TrackerStatus
+
+from homeassistant.components.device_tracker.config_entry import TrackerEntity
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_ENTITIES
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity import DeviceInfo
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from .const import CLIENT, DOMAIN, LOGGER, MDI_ICONS
+
+PARALLEL_UPDATES = 1
+SCAN_INTERVAL = timedelta(seconds=240)
+
+
+async def async_setup_entry(
+    hass: HomeAssistant,
+    entry: ConfigEntry,
+    async_add_entities: AddEntitiesCallback,
+) -> None:
+    """Set up the device_tracker platform."""
+    client: AsyncClient = hass.data[DOMAIN][entry.entry_id][CLIENT]
+    trackers: list[Tracker] = await client.get_trackers()
+
+    entities = [GpsTrackerEntity(client, tracker) for tracker in trackers]
+    hass.data[DOMAIN][entry.entry_id][CONF_ENTITIES].extend(entities)
+    async_add_entities(entities, update_before_add=True)
+
+
+class GpsTrackerEntity(TrackerEntity):
+    """Class for Invoxia™ GPS tracker devices."""
+
+    def __init__(self, client: AsyncClient, tracker: Tracker) -> None:
```

Comment:
I think you should look at `_attr_` shorthand attributes here.

Reply:
Done in 5429434ddec5970aeec5ce97b6e2ba6592e1dc5e